### DISCUSSION TITLE
Binary comparison alignment

### DISCUSSION THREAD

**Discussion by bheston** [WinMerge/winmerge#982] [Q&A]
From what I can tell, the binary comparison goes byte for byte and highlights any differences, but does not detect blocks like the text comparison mode.

For instance, I'm working with images and specifically editing the metadata in the header. When I compare two identical _images_ with different _metadata_ it appears as if the whole file has changed. It would be great if there were similar block detection that could identify that a majority of the content is still the same and line that up. In this example the exact byte offset is not meaningful, so I would expect the need to either not show that or break it into sections somehow.

Is this already a feature I haven't figure out how to turn on?

**Comment by sdottaka** [comment]
Unfortunately, WinMerge's binary comparison cannot detect data insertions or deletions.
